Add Note about how to access Detailed Operations (#1639)
odoo/documentation: content/applications/inventory_and_mrp/inventory/management/incoming/one_step.rst

Now, enter the picking that you want to process. You will be able to
click on *Validate* to complete the move directly as products coming
from suppliers are considered as being always available. 

> Note: if you have Storage Locations activated, you would click the 
> hamburger menu next to the Done quantity to specify the location(s) 
> where you are storing the received product(s).

Reviewer: - [RST cheat sheet: notes admonitions](https://www.odoo.com/documentation/13.0/contributing/documentation/rst_cheat_sheet.html#note)
- RST guidelines: break the line at the 100th character
- not documented yet but we'll use `guilabel` for elements that are part of the interface (fields names, buttons, views etc)

Does this suggestion seem good to you?
```suggestion
.. note::
   If you have :guilabel:`Storage Locations` activated, you can click the hamburger menu next to 
   the :guilabel:`Done quantity` to specify the location(s) where you are storing the received
   product(s).
```
